src/CGAL_Nef_polyhedron.cc

```diff
@@ -94,20 +94,20 @@ PolySet *CGAL_Nef_polyhedron::convertToPolyset() const
 #endif
 
 void CGAL_Nef_polyhedron::resize(const Vector3d &newsize,
```

Comment:
It didn't change in this PR, but is this function signature right? It looks like it's missing a closing parentheses and perhaps more args? (there's a trailing comma)

Reply:
This is just because you're viewing github diffs with a large tab size. The rest of the function parameters are indented to match the parenthesis.

Reply:
-> https://github.com/openscad/openscad/pull/2179/files?ts=2#diff-b626954cb7b549ae78b89800cf925ba7L96

Reply:
Oops, my mistake, I did the same thing the other day...